src/code/z_eff_blure.c

```diff
@@ -771,7 +775,9 @@ void EffectBlure_DrawSimpleVertices(GraphicsContext* gfxCtx, EffectBlure* this,
                 gDPPipeSync(POLY_XLU_DISP++);
             }
 
-            if (1) {} // Necessary to match
+#if OOT_DEBUG
+            if (1) {}
+#endif
```

Comment:
oh no sorry, I didn't even notice this was in a different function. This could be removed by deleting `j` and replacing it with `i * 4` everywhere

Reply:
`4 * i` rather (matches the rest of the function better)